core/amber/src/main/scala/edu/uci/ics/texera/workflow/operators/union/UnionOpDesc.scala

```diff
@@ -0,0 +1,28 @@
+package edu.uci.ics.texera.workflow.operators.union
+
+import com.google.common.base.Preconditions
+import edu.uci.ics.amber.engine.operators.OpExecConfig
+import edu.uci.ics.texera.workflow.common.metadata.{OperatorGroupConstants, OperatorInfo}
+import edu.uci.ics.texera.workflow.common.operators.{OneToOneOpExecConfig, OperatorDescriptor}
+import edu.uci.ics.texera.workflow.common.tuple.schema.Schema
+
+class UnionOpDesc extends OperatorDescriptor {
+
+  override def operatorExecutor: OpExecConfig = {
+    new OneToOneOpExecConfig(this.operatorIdentifier, _ => new UnionOpExec())
+  }
+
+  override def operatorInfo: OperatorInfo = OperatorInfo(
+    "Union",
+    "unions the output rows from multiple input operators",
+    OperatorGroupConstants.UTILITY_GROUP,
+    1, 1, allowMultiInputs = true
```

Comment:
Will it be confusing that we are allowing multiple links to go in one port for some operators, but for other operators the number of in-links in one port is restricted? Should we somehow show that this port accepts multiple inlinks?

Reply:
I agree with you that we should show the port that allow multi-connection differently to the user. Maybe the port can be larger or in a different shape. I remeber in Alteryx or RapidMiner, how they do it is they have a "+" sign below the port, and you can dynamically add more ports, however it's harder to implement.